### PR TITLE
change label for hvdc deletion dialog

### PR DESCRIPTION
**Pull request by EtienneLt**

### CODE REVIEW

File: src/components/dialogs/network-modifications/by-filter-deletion/by-filter-deletion-form.tsx [hunk 1]
```diff
@@ -44,9 +44,17 @@ const ByFilterDeletionForm = () => {
...
+            .filter(
+                (equipmentType) => !equipmentTypesToExclude.has(equipmentType)
+            )
+            .map((element) => {
+                if (element === EQUIPMENT_TYPES.HVDC_LINE) {
+                    return 'Hvdc';
+                } else {
+                    return element;
+                }
+            });
```
Comment (Mathieu-Deharbe) [inline src/components/dialogs/network-modifications/by-filter-deletion/by-filter-deletion-form.tsx:57]:
```suggestion
           .map((equipmentType) =>
                equipmentType === EQUIPMENT_TYPES.HVDC_LINE
                    ? 'Hvdc'
                    : equipmentType
            );
```
Ternary operator is not important, do as you wish. But I think it is better to use the more precise 'equipmentType' than the generic 'element'.

**Review by Mathieu-Deharbe — CHANGES_REQUESTED** [review]
Rectification : this breaks the HVDC filter fetching (when you select 'HVDC' as a filter type and look for HVDC filters in your folders, you can't see them).

I think you didn't only change the label, you changed the wording of the HVDC option used for the back as well.

File: src/components/dialogs/network-modifications/by-filter-deletion/by-filter-deletion-form.tsx (hunk 1 above)
Comment (EtienneLt) [reply src/components/dialogs/network-modifications/by-filter-deletion/by-filter-deletion-form.tsx]: Yes I changed it

**Review by Mathieu-Deharbe — APPROVED** [review]

File: src/hooks/use-get-label-equipment-types.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import { getIdOrValue } from '../components/dialogs/commons/utils';
+
+export default function useGetLabelEquipmentTypes() {
+    const intl = useIntl();
+    return useMemo(
+        () => (equipmentType: string) => {
+            if (equipmentType === EQUIPMENT_TYPES.HVDC_LINE) {
+                return intl.formatMessage({ id: 'Hvdc' });
+            } else {
+                return intl.formatMessage({ id: getIdOrValue(equipmentType) });
```
Comment (thangqp) [inline src/hooks/use-get-label-equipment-types.ts:19]:
```suggestion
                return intl.formatMessage({ id: equipmentType });
```

File: src/hooks/use-get-label-equipment-types.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+            }
+        },
```
Comment (thangqp) [inline src/hooks/use-get-label-equipment-types.ts:21]:
```suggestion
        () => (equipmentType: string) => intl.formatMessage({ id: equipmentType === EQUIPMENT_TYPES.HVDC_LINE ? 'Hvdc' :  equipmentType}),
```

File: src/hooks/use-get-label-equipment-types.ts [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    return useMemo(
+        () => (equipmentType: string) =>
+            intl.formatMessage({
+                id:
+                    equipmentType === EQUIPMENT_TYPES.HVDC_LINE
+                        ? 'Hvdc'
+                        : equipmentType,
+            }),
+        [intl]
+    );
```
Comment (thangqp) [inline src/hooks/use-get-label-equipment-types.ts:22]:
```suggestion
    return useMemo(
        () => (equipmentType: Option) =>
            intl.formatMessage({
                id:
                    equipmentType === EQUIPMENT_TYPES.HVDC_LINE
                        ? 'Hvdc'
                        : getIdOrValue(equipmentType),
            }),
        [intl]
    );
```

File: src/hooks/use-get-label-equipment-types.ts [hunk 5]
```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import { useMemo } from 'react';
+import { useIntl } from 'react-intl';
+import { EQUIPMENT_TYPES } from '../components/utils/equipment-types';
+
```
Comment (thangqp) [inline src/hooks/use-get-label-equipment-types.ts:10]:
```suggestion
import { Option } from '@gridsuite/commons-ui';
import { getIdOrValue } from '../components/dialogs/commons/utils';

```

**Review by thangqp — APPROVED** [review]

**Pull request merged**